### PR TITLE
Add initial `CODEOWNERS` / `.mailmap` files

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Note that patterns are not cumulative. So order is important as only the
+# last matching pattern is used, see
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax
+
+* @oss-review-toolkit/kotlin-devs
```
Comment (mnonnenmacher) [inline .github/CODEOWNERS:5]: GitHub complains that this owner is unknown, is this a false alarm?

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Marcel Bochtler <marcel.bochtler@bosch.io> <git@bochtler.io>
+Martin Nonnenmacher <martin.nonnenmacher@bosch.io> <martin.nonnenmacher@here.com>
```
Comment (mnonnenmacher) [inline .mailmap:2]: Should be "gmail.com" instead of "here.com".

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (sschuberth) [reply .github/CODEOWNERS:5]: There probably was some glitch, as previously I also could not choose `@oss-review-toolkit/kotlin-devs` manually from the "Reviewers" menu, but it seems to be resolved now. Let's just see if this works once merged.

Comment (sschuberth) [reply .github/CODEOWNERS:5]: Correction, there indeed was a permission issue that I just fixed.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**